src/Component.js

```diff
@@ -41,11 +39,11 @@ export class CopyToClipboard extends React.PureComponent {
       text: _text,
       onCopy: _onCopy,
       options: _options,
+      onClickCapture,  // eslint-disable-line
```

Comment:
I'm actually not sure what's setting it, but it wasn't explicitly set in my tests and yet it had a value.